workloads/storage/pvcscale_README.md

```diff
@@ -0,0 +1,116 @@
+### Purpose
+PVC scale test is designed to create desired number of pods and attach PVC to every pod from storage backend dynamically 
+using storage storage class. 
+
+### Prerequesties
+- PVC test requieres functional storage backend and storage class. 
+- Pbench [web server](https://distributed-system-analysis.github.io/pbench/doc/server/pbench-web-server.html)
+
+### Environment variables for PVC Scale workload playbook
+
+```
+###############################################################################
+# Ansible SSH variables.
+###############################################################################
+export PUBLIC_KEY
```

Comment:
I'd prefer to drop all of the exports even though they're necessary.  Folks should know this and they're not interested in `export` keyword, they're interested in the variable itself.